packages/medusa/src/utils/currencies.ts

```diff
@@ -675,6 +675,15 @@ export const currencies: Record<string, Currency> = {
     code: "MUR",
     name_plural: "Mauritian rupees",
   },
+  MVR: {
+    symbol: "MVR",
+    name: "Maldivian Rufiyaa",
+    symbol_native: "ރ",
+    decimal_digits: 0,
```

Comment:
question: Most documentation says it has 2 decimals. Can I get you to check up on that?

Reply:
you are right!. updated now